fhir-examples/src/main/resources/json/ibm/basic/BasicReference.json

```diff
@@ -49,25 +49,25 @@
       }
     },
     {
-      "url": "http://example.org/ReferenceId",
+      "url": "http://example.org/Reference-id",
```

Comment:
is this a pattern I should follow if I run into extensions?

Reply:
doesn't really matter; just needs to match whats in the corresponding SearchParameter in extension-search-parameters.json